tests/integration/enums/python/__main__.py

```diff
@@ -14,15 +14,9 @@ class Farm(str, Enum):
     PLANTS_R_US = "Plants'R'Us"
     PULUMI_PLANTERS_INC = "Pulumi Planters Inc."
 
-
-current_id = 0
-
-
 class PlantProvider(ResourceProvider):
     def create(self, inputs):
-        global current_id
-        current_id += 1
-        return CreateResult(str(current_id), inputs)
```

Comment:
Does this also fix the `TestDynamicPython` test? How?

Reply:
Both tests use this Python program.